src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -70,6 +70,12 @@ internal class BuildCommand : CommandBase
             Description = "Additional information used by this tool to generate and run build scripts.")]
         public string[] Properties { get; set; }
 
+        [Option(
+            OptionTemplates.ManifestFileDir,
+            CommandOptionType.SingleValue,
+            Description = "The path to a directory where build manifest file is to be put into.")]
+        public string ManifestFileDir { get; set; }
```

Comment:
Annotate with `DirectoryExists`

Reply:
I cannot do it here because we can create the directory if it does not already exist. It is just a matter of convenience to the end user.